.github/workflows/pull_request.yml

```diff
@@ -101,21 +101,3 @@ jobs:
             test-reports/fdb-relational-jdbc/
             test-reports/fdb-relational-server/
             test-reports/yaml-tests/
-      - name: Test Coverage Comment
```

Comment:
Post #3122, I think we need to remove the equivalent step from `core-tests` above as well.